src/app/shared/services/record.service.ts

```diff
@@ -20,19 +20,26 @@
  * as an Intergovernmental Organization or submit itself to any jurisdiction.
 */
 
-import { Injectable } from '@angular/core';
+import { Injectable, Inject } from '@angular/core';
 import { Http } from '@angular/http';
 import { Observable } from 'rxjs/Observable';
 import 'rxjs/add/operator/map';
+import { APP_CONFIG } from '../../app.config';
 
 @Injectable()
 export class RecordService {
+  private type: string;
```

Comment:
To be consistent with the way things are named in Invenio, please rename `type` -> `pid_type`, `recid`, `pid_value`